src/pages/index.tsx

```diff
@@ -56,20 +59,22 @@ export default function Home() {
           sx={{
             display: "flex",
             alignItems: "center",
-            width: "100%",
             flexDirection: "column",
             padding: "30px",
           }}
         >
-          {posts?.map((item, index) => (
-            <Box width="60vw" key={index}>
-              <CommonPostCard
-                post={item}
-                userId={userStatus.user?.userId}
-              />
-            </Box>
-          ))}
+          <SearchPanel setPosts={setPosts} />
+          {posts.length > 0 ? (
+            posts.map((item, index) => (
+              <Box width="60vw" key={index}>
+                <CommonPostCard post={item} userId={userStatus.user?.userId} />
+              </Box>
+            ))
+          ) : (
+            <Typography>No post found</Typography>
```

Comment:
![image](https://user-images.githubusercontent.com/69895436/228533241-ae685f29-fb8a-4fa1-85de-f5ff81976a29.png)
The message doesn't go the same way as design
![image](https://user-images.githubusercontent.com/69895436/228533136-bbf7ce91-bf84-4375-8297-0eb69c8ccccf.png)

Reply:
Done

Reply:
![image](https://user-images.githubusercontent.com/69895436/228572323-86982134-45fd-4233-aa98-06186493b940.png)
it should on the center, currently it's on top

Reply:
just make it look like approximately is fine tho

Reply:
Fixed it